cmd/create/operatorroles/cmd.go

```diff
@@ -286,15 +312,51 @@ func run(cmd *cobra.Command, argv []string) {
 		reporter.Errorf("Invalid mode. Allowed values are %s", modes)
 		os.Exit(1)
 	}
+
+	operatorIAMRoleList := []ocm.OperatorIAMRole{}
+	for _, operator := range aws.CredentialRequests {
+		operatorIAMRoleList = append(operatorIAMRoleList, ocm.OperatorIAMRole{
+			Name:      operator.Name,
+			Namespace: operator.Namespace,
+			RoleARN:   getOperatorRoleArn(operatorRolesPrefix, operator, creator),
+		})
+	}
+	clusterConfig := ocm.Spec{
+		OperatorIAMRoles: operatorIAMRoleList,
+	}
+	err = ocmClient.UpdateCluster(clusterKey, creator, clusterConfig)
+	if err != nil {
+		reporter.Errorf("Error updating the cluster with the operator roles %v", err)
+	}
```

Comment:
I'm not sure I agree with this change. I know of at least 2 customers that are relying solely on the documentation to create the operator roles and OIDC provider. If we cannot automatically detect those in OCM, we need to _at least_ send the prefix to OCM during 'create cluster' (we at least have `iam:ListRoles` and `iam:GetRole` permissions in the installer role).

I think patching the cluster from here would be a blocker for them, unfortunately